translations/translations-workflow.md

```diff
@@ -24,72 +23,46 @@ as slug), with many *components*:
-be amended**.
-
-For the synchronisation workflow, it's semi automatic:
-
-- A GitHub application is installed in the RERO GitHub organization.
-- In each repository, a web hook is configured, as [explained in the weblate
-  documentation][1].
-
-This ensures two things:
-
-1. As changes are merged into the `dev` branch of our repository, the weblate
-   service is informed through the webhook and pulls changes to the weblate
-   `git` repository. It's done with a `git rebase` command, as it can be
-   configured in the component settings [component/settings/version control].
-1. As translations are being made through the weblate interface, commits are
-   being made. 24 hours after a commit ([see settings section][5]), a PR is
-   created on the corresponding GitHub repository (called *upstream* by the
-   weblate documentation).
-
-As it is managed with `git`, and as we use *bilingual* files (each translation
-files has two languages), which supposes to update the catalogs, conflicts can
-happen. To avoid such conflicts, [the documentation][2] proposes something like
-the following:
-
-*You'll need to [install and configure the `wlc` client][4]*.
-
-1. The `dev` branch (*upstream*) is moving. Developers do not
-   `extract_messages` nor `update_catalog`.
-1. From time to time, the person responsible for the translations updates
-   weblate:
-    1. Ensure weblate is in a clean state: all changes are commited (`wlc
-       commit rero_plus/<component>`).
-    1. Block the translations on the component (`wlc lock
-       rero_plus/<component>`).
-    1. Push weblate on `rero:dev` (`wlc push rero_plus/<component>`).
-    1. Pull the weblate commit locally (`git pull`).
-    1. Extract the messages and update the catalogs. Commit the changes.
-    1. Push *upstream* (`git push`).
-    1. Verify that weblate is being automatically updated.
-    1. Unlock translations on weblate (`wlc unlock rero_plus/<component>`).
-1. Translators can work on weblate.
-1. Weblate creates PR on the `dev` branch of the GitHub repository. 
-1. Weblate PR are being merged into `dev`.
-
-The section 2 could be scripted, once it's validated.
+**We do not use the automatic workflow through the GitHub webhook, to avoid
+conflicts.**
+
+The starting situation is just after a release: the GitHub `translations` branch
+is on the same commit than the [weblate]][12] `translations` branch. Also, the
+GitHub `translations` branch is on the same commit than the GitHub `dev`
+branch.
+
+1. Some changes are done on the `dev` branch.
+1. Some changes are done on weblate.
+1. Ensure that on weblate, all changes is committed.
+1. Lock the component.
+1. Then ensure that on GitHub all translations pull requests are merged into
+   `translations` branch.
+1. Fetch and pull new upstream commits (`translations` branch) on your local
+   `translations` branch.
+1. Maybe, you want to squash several commits into one.
```

Comment:
Maybe this can be done after messages extraction, to have only one commit for the new translations on `dev`?

Reply:
You're right. I've added the line 48 to commit the extracted strings into the already existing commit. I'm not extra confident, but I prefer extracting and updating catalog *after* squashing, because sometimes I got conflicts, doing so.

Reply:
👍 At the end, the goal is to have only one commit being merged in `dev` branch.